docs/2.7.0/bin/setup-sphinx-source-tree

```diff
@@ -16,26 +16,22 @@ prefix=$(basename $(realpath $DIR/..))
 
 mkdir -p $SPHINX_DIR/source/canton $SPHINX_DIR/source/daml-finance
 tar xf $DOWNLOAD_DIR/sphinx-source-tree-$RELEASE_TAG.tar.gz -C $SPHINX_DIR --strip-components=1
-tar xf $DOWNLOAD_DIR/canton-doc-sources-$CANTON_RELEASE_TAG.tar.gz -C $SPHINX_DIR/source/canton
-rm $SPHINX_DIR/source/canton/canton-refs.rst
+tar xf $DOWNLOAD_DIR/canton-doc-sources-$CANTON_RELEASE_TAG.tar.gz -C $SPHINX_DIR/source/canton ./includes # only extract the includes folder
```

Comment:
ℹ️ The current canton documentation "assembly" package (`tar.gz`) may still contain all documentation files.

However, we only want to have the `includes` folder which contains
* generated files: Canton Console Commands, Error Code, Metrics, and Versioning table
* mirror files: Canton `.conf`, Scala source code `.scala`, and other verbatim files which are referenced by `literalincludes`
* snippet output data: The data which is required to replace the snippet directives in the `.rst` files